tests/util.py

```diff
@@ -1,8 +1,7 @@
 import io
 from collections.abc import Sequence, Mapping
-from operator import itemgetter
 from pathlib import Path
-from typing import Any
+from typing import Any, Callable
```

Comment:
https://docs.python.org/3/library/typing.html#typing.Callable

Deprecated since version 3.9: [collections.abc.Callable](https://docs.python.org/3/library/collections.abc.html#collections.abc.Callable) now supports subscripting ([]).

Reply:
\+